changelog/5.5.0_2022-06-20/enhancement-update-sdk

```diff
@@ -1,6 +1,6 @@
 Enhancement: Update SDK
 
-We've updated the ownCloud SDK to version 3.0.0-alpha.10.
+We've updated the ownCloud SDK to version 3.0.0-alpha.13.
```

Comment:
We already have `3.0.0-alpha.17` in current master and the changelog item has been moved to the archive of a released version of `web`. Please drop the changelog item changes.

Reply:
Ah lol, you already did :D

Reply:
Yeah, sure - wanted to figure out first, why the original PR was broken. "Unfortunately" it's not broken anymore apparently 🤷🏻